sql3/planner/oppqldistinctscan.go

```diff
@@ -265,26 +265,18 @@ func (i *distinctScanRowIter) Next(ctx context.Context) (types.Row, error) {
 			row[0] = pql.NewDecimal(val, t.Scale)
 
 		case *parser.DataTypeIDSet:
-			val, ok := result.([]uint64)
+			val, ok := result.(int64)
```

Comment:
I don't actually know of anything wrong here but this feels like a surprising change in semantics. Or is this because we're changing the semantics of distinct or something?

Reply:
Yeah - the original code was never hit, and when it was result was not as we expected.